optimum_benchmark/scenarios/energy_star/scenario.py

```diff
@@ -7,7 +7,7 @@
 from tqdm import tqdm
```

Comment:
better not change anything in energy star, the energy start PR will take care of it, otherwise will have conflicts